suite-native/storage/src/storage.ts

```diff
@@ -24,14 +27,17 @@ const retrieveStorageEncryptionKey = async () => {
     return secureKey;
 };
 
-export const initMmkvStorage = async (): Promise<Storage> => {
```

Comment:
nit: not sure if there is benefit of spliting this to two functions since it again used only here

Reply:
fixed ec7bdb40a